web/src/settings_config.ts

```diff
@@ -1193,15 +1193,15 @@ export const stream_privacy_policy_values = {
         code: "invite-only-public-history",
         name: $t({defaultMessage: "Private, shared history"}),
         description: $t({
-            defaultMessage: "Joining and viewing messages requires being added by a subscriber",
+            defaultMessage: "Joining and viewing messages requires being invited",
         }),
     },
     private: {
         code: "invite-only",
         name: $t({defaultMessage: "Private, protected history"}),
         description: $t({
             defaultMessage:
-                "Joining and viewing messages requires being added by a subscriber; new subscribers cannot see messages sent before they joined",
+                "Joining and viewing messages requires being invited; new subscribers cannot see messages sent before they got access",
```

Comment:
So you have to be actually subscribed to see any newly sent messages in protected. Changed to:

> Joining and viewing messages requires being invited; users can only view messages sent while they were subscribed